### ISSUE TITLE
NSPOSIXErrorDomain Code=12 "The operation couldn’t be completed. Cannot allocate memory"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [willmyers428/google-email-uploader-mac#12]
```
What steps will reproduce the problem?
1. Begin upload with Mail.app source
2. Wait
3. Start seeing errors


What is the expected output? What do you see instead?

I'm expecting it to work.  Instead I'm seeing a memory error:


Error Domain=NSCocoaErrorDomain Code=256 UserInfo=0x34ca40 "The file 
“773003.emlx” couldn’t 
be opened." Underlying Error=(Error Domain=NSPOSIXErrorDomain Code=12 "The 
operation couldn’t 
be completed. Cannot allocate memory")

/Users/dfbills/Library/Mail/Mailboxes/MW News/2008.mbox/Messages/773003.emlx


What version of the product are you using? On what operating system?

Version 1.1.1 on MacOS 10.6.2


Please provide any additional information below.

This is a new error for me in this build of the uploader.
```

Original issue reported on code.google.com by `dfbi...@gmail.com` on 25 Feb 2010 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Is there a file with a name like "773003.1.emlxpart" in the folder 
Library/Mail/Mailboxes/MW News/2008.mbox/Messages? If so, how big is it?
```

Original comment by `gregrobbins` on 25 Feb 2010 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
Yes, the file is present(and the attachment is visible in quicklook.  The file 
size is 4.2MB.
```

Original comment by `dfbi...@gmail.com` on 25 Feb 2010 at 4:04

**Comment by GoogleCodeExporter** [comment]
```
Would it be helpful to have the .emlx file?
```

Original comment by `dfbi...@gmail.com` on 25 Feb 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
if it can be repro'd with an emlx file (and maybe associated emlxpart files) 
yes, it would be very helpful to have 
those.

What kind of mail archive is this? (POP or IMAP)

How much memory does the Mac have?
```

Original comment by `gregrobbins` on 25 Feb 2010 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
I've attached the file.

My Mac has 8GB of ram and is a MacPro 2 x 228 Ghz Quad-Core.
```

Original comment by `dfbi...@gmail.com` on 25 Feb 2010 at 10:08

Attachments:
- [773003.emlx](https://storage.googleapis.com/google-code-attachments/google-email-uploader-mac/issue-12/comment-5/773003.emlx)

**Comment by GoogleCodeExporter** [comment]
```
Also, this is locally stored mail from an Exchange2007 account. (Not actually 
stored on the server.)
```

Original comment by `dfbi...@gmail.com` on 25 Feb 2010 at 10:09

**Comment by GoogleCodeExporter** [comment]
```
I cannot reproduce this. The email message uploads without problem for me.

Where is the error message appearing? In a dialog in the application, in the 
system console window, 
somewhere else? 

Does the uploader application crash, stop uploading, or continue to upload 
other mail?

Is this message really failing to upload, or is the error message appearing 
despite the message actually 
uploading correctly?

```

Original comment by `gregrobbins` on 27 Feb 2010 at 2:11

**Comment by GoogleCodeExporter** [comment]
```
I was seeing it over and over in the application log.  I didn't check the 
console.  

The uploader seemed to go on with its work, but there were so many of these 
messages that I was concerned.

The message definitely hasn't been uploaded.  I can run the process again and 
try to capture the full log.
```

Original comment by `dfbi...@gmail.com` on 27 Feb 2010 at 2:22

**Comment by GoogleCodeExporter** [comment]
```
Fixed a leak of emlx message body text:
http://code.google.com/p/google-email-uploader-mac/source/detail?r=31

Throttle uploads when currently uploading message size exceeds 10 MB:
http://code.google.com/p/google-email-uploader-mac/source/detail?r=30

```

Original comment by `gregrobbins` on 22 Mar 2010 at 5:57
- Changed state: **Started**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed now in the 1.1.2 release, hopefully. Thanks for the issue report.
```

Original comment by `gregrobbins` on 30 Mar 2010 at 9:31
- Changed state: **Fixed**
